src/bounded/sat.rs

```diff
@@ -0,0 +1,811 @@
+// Copyright 2022-2023 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+//! A bounded model checker for flyvy programs using an SAT solver.
```

Comment:
I think this comment should explain that it checks using both cadical and kissat and checks that the results are the same.

There's also a small typo (an -> a).

Reply:
Using both cadical and kissat isn't really a feature, we just haven't decided which one to use yet. We'll pick one in the future once we have benchmarking.